### ISSUE TITLE
Adjust New Issue link

### ISSUE THREAD

**Issue by klemmster** [klemmster/libavg_svn_migration#9]
Should point to https://github.com/libavg/libavg/issues/new

depends on #21

**Comment by klemmster** [comment]
same as #13 and #8 
I guess I'll have to write an redmine plugin for that. Right now the issues links reside here:
https://www.libavg.de/site/projects/libavg/wiki/BugReport

**Comment by uzadow** [comment]
Or just put these things in the Menu on the left? Maybe underneath 'download'?
